### PR TITLE
#40 tech(fetch.js) Function to fetch movie-API from json-file.

### PR DESCRIPTION
**Pull request by Patseroni**
## Pull Request Type
Please check the type of change your PR introduces:

- [x] Feature (a new feature for the project)
- [ ] Fix (a bug fix)
- [ ] Chore (updating grunt tasks etc; no production code change)
- [ ] Refactor (refactoring production code)
- [ ] Docs (changes to the documentation)
- [ ] Style (formatting, missing semi colons, etc; no production code change)
- [ ] Test (adding missing tests, refactoring tests; no production code change)

## Description
Function to be able to fetch all information about the movies from the json-file.

## Motivation and Context
[Why is this change required? What problem does it solve?]

## How Has This Been Tested?
When console.logging it shows everything correctly for me when the page has loaded.

## Screenshots (if appropriate):
[Add screenshots here]

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have named this PR according to following naming convention: #[issue-number] PR description

## Additional Information:
[Add any other information about the PR here]

## Related Issues:
[Link any related issues here]

### CODE REVIEW

**Review by Deansie — APPROVED** [review]
Looks good, movies appear in the console. Well done! Looking forward to get this function implemented into the movie cards. Sort the conflict and merge once you get the second approval.

**Review by AlexCode-dot — APPROVED** [review]
Looks good! I approve

**Review by Fagerdahl — APPROVED** [review]
I approve, all is good in the hood :-) Now merge this Patrik B-)

**Pull request merged**
